chromium_src/net/base/network_isolation_key.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_NET_BASE_NETWORK_ISOLATION_KEY_H_
+#define BRAVE_CHROMIUM_SRC_NET_BASE_NETWORK_ISOLATION_KEY_H_
+
+#define BRAVE_NETWORK_ISOLATION_KEY_H \
```

Comment:
can we do 
```
#define GetTopFrameOrigin             \
  GetEffectiveTopFrameOrigin() const; \
  base::Optional<url::Origin>& GetTopFrameOrigin
```
so that we don't need any patches into header and put the implementation in .cc override

Reply:
Good point, I do certainly prefer not to introduce a .patch file if not strictly needed. Fixed!